class-two-factor-core.php

```diff
@@ -192,43 +207,28 @@ public static function uninstall() {
 	 *
 	 * @return array List of provider keys and paths to class files.
 	 */
-	public static function get_providers_registered() {
```

Comment:
TODO: consider that an integration could have been using this already. Might need to keep the name or add a helper method.

Reply:
This was added only in the previous version so we don't have to worry about that.